scripts/myrial

```diff
@@ -143,6 +146,12 @@ def main(args):
                 raise "Options dot and --plan are incompatible"
 
             print operator_to_dot(processor.get_logical_plan())
+        elif opt.opt_dot:
+            if opt.from_repr:
```

Comment:
Sorry Brandon, I know what you mean by taking in a plan from repr form, as opposed to generating a plan from parsing a MyriaL program.  But I can't get RACO to work with repr's. 

Is this correct?

```bash
myrial -r q9.myl > q9.plan
myrial -d --plan "q9.plan"
```